schedule/yast/autoyast/autoyast_lvm.yaml

```diff
@@ -9,7 +9,6 @@ description: >
      - Passwords are set for the users in the profile
 vars:
   AUTOYAST: autoyast_sle15/autoyast_lvm.xml
-  AUTOYAST_VERIFY: autoyast_sle15/autoyast_lvm.list
```

Comment:
We should also remove file and scripts, unless they are used in tftp test, which I currently work on.

Reply:
I noticed the AUTOYAST_VERIFY being used in lib/main_common.pm . Maybe we should remove that whole chunk of code as well, if no one is using it.